doc/source/installation.rst

```diff
@@ -1,140 +1,150 @@
-If you plan to use the parallel OpenMP algorithms, you need to
-install MDAnalysis with pip and have a working OpenMP installation.
+The latest version of **MDAnalysis** can be installed using :ref:`conda-installation` (recommended), 
+or :ref:`pip-installation`. For development and contribution users can :ref:`source-installation`.
+ 
+Currently, the `conda`_ releases only support serial calculations. If you plan to use 
+the parallel `OpenMP`_ algorithms, you need to install MDAnalysis with `pip`_ and have a 
+working `OpenMP`_ installation.
 
-MDAnalysis has a separate :ref:`test suite <mdanalysistests>` **MDAnalysisTests** that is required to run the test cases and examples.
-The test files change less frequently, take up around 90 MB of space,
-and are not needed for daily use of MDAnalysis. However, they are often used in examples,
-including many in this User Guide. If you are not interested in developing
-MDAnalysis or using the example files, you most likely don't need the tests. If you want to
-run examples in the User Guide, install the tests.
-The tests are distributed separately from the main package.
+**MDAnalysisTests** is optional and it is a separate :ref:`test suite <mdanalysistests>` used 
+for verifying MDAnalysis installations and running User Guide examples. It is not required for 
+daily use, but some tutorials depend on it. The package is 90 MB and does not change frequently. 
+If you plan to use tutorial examples or contribute to MDAnalysis development, installing it is recommended.
 
-.. note::
-
-    If you are installing on Windows, you must have
-    Microsoft Visual C++ 14.0 installed. If your installation
-    fails with the error message:
+MDAnalysis requires Python 3.8 or later. Ensure you have an appropriate Python version installed before proceeding.
 
-        error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio": https://visualstudio.microsoft.com/downloads/
-
-    Try installing Build Tools for Visual Studio from
-    https://visualstudio.microsoft.com/downloads/ (scroll
-    down to the Tools for Visual Studio section).
+.. note::
+    MDAnalysis supports **Linux**, **macOS**, and **Windows** (Python 3.8+). 
 
+    - If you encounter errors on **Windows** related to **Microsoft Visual C++ 14.0**, install the required **Build Tools for Visual Studio** from: `Microsoft Visual Studio Downloads`_. 
+    - If you encounter any other issues following these instructions, seek help on `GitHub Discussions (Installation)`_.    
 
-If you encounter any issues following these instructions, please
-ask for help on `GitHub Discussions (Installation)`_.
+.. warning::
+    `conda`_ and `pip`_ installations **do not include** external programs such as `HOLE`_.
 
-.. _`GitHub Discussions (Installation)`: https://github.com/MDAnalysis/mdanalysis/discussions/categories/installation
+.. _conda-installation:
 
-conda
-=====
+conda 
```

Comment:
Call it mamba/conda or Installation with conda-forge packages ... consistent with what we're doing elsewhere.